examples/runners/wgpu/src/lib.rs

```diff
@@ -11,6 +11,13 @@ pub enum RustGPUShader {
     Compute,
 }
 
+#[allow(dead_code)]
+pub struct ShaderConstants {
```

Comment:
Instead of redefining `ShaderConstants` couldn't the `wgpu` and `ash` runners pull in the shader as a crate dependency like `cpu` does? That would remove a quite a bit of duplicate code.

Reply:
It could but in the future it would be super annoying. Any other shader would have to include the sky shader just to get that struct. (I intend to use it for other shaders as well, similar to shadertoy). Could create yet another package as a "shared code" thing but I also don't really like that sound of that if its just a single struct.

Reply:
if we end up with more shared code that would be the way to go though adding a "shared code" crate.